### PR TITLE
Adding the load & chaos testing video

### PR DESCRIPTION
**Pull request by msimecek**
Based on the recommendations from the content dev team, placing the video near the top of the page, where it makes most sense (to me at least :) ). Happy to move it around, but I thougt that since it should be H2 in order to show up in ToC, this is a good place - before getting into design areas and recommendations.

Also added a new link to the main page. Hope it works, not sure how to test it :)

### CODE REVIEW

**Comment by PRMerger13** [comment]
@msimecek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: well-architected/mission-critical/index.yml [hunk 1]
```diff
@@ -66,3 +66,11 @@ landingContent:
             url: https://github.com/azure/mission-critical-online
           - text: Mission-Critical Connected
             url: https://github.com/azure/mission-critical-connected
+
+ # Card
+  - title: Video library
+    linkLists:
+      - linkListType: video
+        links:
+          - text: Resiliency with Azure Load Test and Azure Chaos Studio
```
Comment (CalCof) [inline well-architected/mission-critical/index.yml:75]: Can we change the text ("Resiliency with Azure Load Test and Azure Chaos Studio") to read: "Continuous Validation with Azure Load Test and Azure Chaos Studio"

The url will also need changed as well

Comment (msimecek) [reply well-architected/mission-critical/index.yml:75]: Sure can!

File: well-architected/mission-critical/mission-critical-deployment-testing.md [hunk 2]
```diff
@@ -37,6 +37,10 @@ This design area focuses on how to eradicate downtime and maintain application h
 >
 > The [reference implementations](mission-critical-overview.md#illustrative-examples) are part of an open source project available on GitHub. The code assets illustrate considerations and recommendations for acheiving optimal CI/CD pipelines for a mission-critical application.
 
+## Demo: End-to-end resiliency testing with Azure Load Test and Azure Chaos Studio
```
Comment (CalCof) [inline well-architected/mission-critical/mission-critical-deployment-testing.md:40]: Can we change the text ("End-to-end resiliency testing with Azure Load Test and Azure Chaos Studio") to read: "Continuous validation and testing with Azure Load Test and Azure Chaos Studio"

**Review by CalCof — CHANGES_REQUESTED** [review]
Minor positioning tweaks, then should be good to go. Thanks Martin.

**Comment by PRMerger19** [comment]
@msimecek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@msimecek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by CalCof** [comment]
LGTM

**Comment by CalCof** [comment]
#sign-off

**Comment by Court72** [comment]
I staged the changes in the private repo: https://github.com/MicrosoftDocs/well-architected-pr/pull/211.
They rendered as expected and passed validation.
Merging this PR now.
Thanks!

**Review by Court72 — APPROVED** [review]

**Pull request merged**